### PR TITLE
Add linelist

### PR DESCRIPTION
**Pull request by zkamvar**
This adds the infrastructure for using the linelist package.

At the moment, the linelist package is a fancy aliaser. In the future, it will fully implement hxl searches and checking.

### CODE REVIEW

File: inst/rmarkdown/templates/outbreak/skeleton/skeleton.Rmd [hunk 1]
```diff
@@ -455,9 +491,9 @@ mortality_rate(deaths, population, multiplier = 10^4) %>%
 #### 2x2 tables
 
 ```{r two_by_two, message = FALSE, warning = FALSE}
-outcome <- linelist_cleaned$outcome == "Death"
-is_male <- linelist_cleaned$sex == "Male"
-is_child <- as.integer(linelist_cleaned$age) <= 12
+outcome <- get_epivars(linelist_cleaned, "outcome")== "Death"
+is_male <- gender(linelist_cleaned) == "Male"
+is_child <- as.integer(as.character(age(linelist_cleaned))) <= 12
```
Comment (zkamvar) [inline inst/rmarkdown/templates/outbreak/skeleton/skeleton.Rmd]: Note: this is a common mistake that leads to incorrect results silently. 

``` r
x <- factor(sample(1000:10000, 10))
as.integer(x)
#>  [1]  1  2  9  6  3 10  8  4  5  7
as.integer(as.character(x))
#>  [1] 3788 4575 8662 6015 4721 8889 7273 4888 5813 6140
```

<sup>Created on 2019-01-30 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

CC @aspina7, @dirkschumacher

Comment (dirkschumacher) [reply inst/rmarkdown/templates/outbreak/skeleton/skeleton.Rmd]: Was age a factor in this case?

**Comment by zkamvar** [comment]
It was indeed.

**Comment by zkamvar** [comment]
Closing this as stale

**Pull request closed without merging**